src/Microsoft.Extensions.Configuration.AzureAppConfiguration/TracingUtils.cs

```diff
@@ -103,6 +103,7 @@ private static string CreateCorrelationContextHeader(RequestType requestType, Re
             IList<string> correlationContextTags = new List<string>();
             
             correlationContextKeyValues.Add(new KeyValuePair<string, string>(RequestTracingConstants.RequestTypeKey, Enum.GetName(typeof(RequestType), requestType)));
+            correlationContextKeyValues.Add(new KeyValuePair<string, string>(RequestTracingConstants.ReplicaCount, requestTracingOptions.ReplicaCount.ToString()));
```

Comment:
How about adding replica count only if the app is using more than 1 endpoint? By default, we know that an app always uses at least 1 endpoint.

Reply:
I think it is a good suggestion and saves us space in the header when only one endpoint is available.

Reply:
Done!